.github/workflows/nightly.yaml

```diff
@@ -25,6 +31,14 @@ jobs:
       - run: mvn -version
       - name: Unit Tests
         run: mvn install --errors --batch-mode --no-transfer-progress -Dcheckstyle.skip -T 1C
+      - name: Create issue
+        if: ${{ failure() }}
```

Comment:
Where is this defined?

Reply:
I think it is default. Any failure in this job would make it true.

Reply:
Thank you. This is the document https://docs.github.com/en/actions/learn-github-actions/expressions#failure.